pkg/test/client.go

```diff
@@ -38,6 +39,10 @@ type FrameworkClient interface {
 	Update(gCtx goctx.Context, obj runtime.Object) error
 }
 
+func retryOnAnyError(err error) bool {
+	return true
```

Comment:
Not sure if it's possible to get a not found error on delete, but we'd probably be okay with that error right?
```suggestion
	return !apierrors.IsNotFound(err)
```